### ISSUE TITLE
Inpainting broken

### ISSUE THREAD

**Issue by lhucklen** [Skquark/structured-prompt-generator#3]
![40E5327C-B40E-40A4-906A-22022BE2B507](https://user-images.githubusercontent.com/74945062/196452416-43bc2ba7-6a16-4b4e-b86a-2d6f5ccf5879.png)

**Comment by Skquark** [comment]
That should be fixed now, thanks for pointing out.  HuggingFace has been making some breaking changes on diffusers lately and I've been focusing the new stuff on a whole new version of this project with a better Flutter WebUI that does the same but better, you're gonna probably want to switch to my Stable Diffusion Deluxe Edition to be made public in just a few more days.
By the way, there's a new HuggingFace Inpainting Model Card that you'll also need to accept for it to work (as of 2 days ago), the link to it I just added to the api instructions, so make sure to accept that real quick..

**Comment by lhucklen** [comment]
Any headway on that deluxe  edition?

**Comment by Skquark** [comment]
Yup, finally got the Deluxe Edition working well and looking beautiful.  Still ironing out minor enhancements, but from all my testing it feels ready.  You're gonna love it.... I haven't announced it public yet, but I think I can call it out of beta now.. Here's the link for **Stable Diffusion Deluxe** - https://colab.research.google.com/github/Skquark/AI-Friends/blob/main/Stable_Diffusion_Deluxe.ipynb
All feedback or contributions appreciated, you'll be one of the first to try it out.
